android-sdk/src/main/java/ai/evolv/android_sdk/Execution.java

```diff
@@ -34,21 +34,7 @@ String getKey() {
     }
 
     void executeWithAllocation(JsonArray rawAllocations) throws EvolvKeyError {
```

Comment:
Is this an empty function now? If so please add TODO to clean up

Reply:
done
tnx